src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -122,15 +151,17 @@ public Uri SourceUri
             string source,
             string apiKey,
             bool noServiceEndpoint,
+            SymbolPackageUpdateResourceV3 symbolPackageUpdateResource,
             TimeSpan requestTimeout,
             ILogger log,
             CancellationToken token)
         {
+            var isSymbolEndpointSnupkgCapable = symbolPackageUpdateResource != null;
```

Comment:
are we only pushing `snupkgs` now? or do we still allow pushing `symbol.nupkg`?

Reply:
we allow both.  
to push symbol.nupkg , you need to explicitly specify symbolSoruce in which case symbolPackageUpdateResource will always be null since we will not initialize it.